edit_course/operations/configure.py

```diff
@@ -211,15 +211,15 @@ def configure_content(instance, url):
         url = url.strip()
         response = requests.get(url)
     except Exception as e:
-        return [_("Request failed: {}").format(str(e))]
+        return [_("Request failed: {}.").format(str(e))]
 
     instance.configure_url = url
     instance.save()
 
     try:
         config = json.loads(response.text)
     except Exception as e:
-        return [_("Failed to parse the JSON: {}").format(str(e))]
+        return [_("Failed to parse the JSON: {}.").format(str(e))]
```

Comment:
`_("JSON parser raised an error '{error!s}', and thus configuration of a course was aborted.".format(error=e)`